relayer/spy_relayer/src/__tests__/utils/swim.ts

```diff
@@ -63,6 +63,7 @@ test("parseTransferWithPoolPayload", () => {
     targetChainRecipient: convertAddressToUint8(targetAddress, CHAIN_ID_SOLANA),
```

Comment:
this should almost certainly be called `convertAddressToUint8Array`. Arguably it can't be anything but an array because turning an address into a single byte is absurd, but it actually gave me pause for a second because I thought "maybe he's trying to derive the Wormhole Chain Id from the address" until I realized that even that would be 2 bytes and not 1...